`css` build broken after #3215

After https://github.com/GaloisInc/saw-script/pull/3215, `css` fails to build:

```
[1 of 2] Compiling Main             ( saw-tools/css/Main.hs, interpreted ) [Source file changed]

saw-tools/css/Main.hs:103:16: error: [GHC-83865]
    • Couldn't match expected type: IO C.CryptolEnv
                  with actual type: Maybe Cryptol.Parser.AST.ImportSpec
                                    -> IO C.CryptolEnv
    • Probable cause: ‘C.importCryptolModule’ is applied to too few arguments
      In a stmt of a 'do' block:
        cryenv' <- C.importCryptolModule
                     sc cryenv (Left inputModule) Nothing C.PublicAndPrivate Nothing
      In the expression:
        do let out = ...
           sc <- mkSharedContext
           C.scLoadPreludeModule sc
           C.scLoadCryptolModule sc
           ....
      In an equation for ‘cssMain’:
          cssMain css [inputModule, name]
            | cssMode css == NormalMode
            = do let out = ...
                 sc <- mkSharedContext
                 C.scLoadPreludeModule sc
                 ....
    |
103 |     cryenv' <- C.importCryptolModule sc cryenv (Left inputModule)
    |                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...

saw-tools/css/Main.hs:104:29: error: [GHC-83865]
    • Couldn't match expected type ‘Bool’
                  with actual type ‘C.ImportVisibility’
    • In the fifth argument of ‘C.importCryptolModule’, namely
        ‘C.PublicAndPrivate’
      In a stmt of a 'do' block:
        cryenv' <- C.importCryptolModule
                     sc cryenv (Left inputModule) Nothing C.PublicAndPrivate Nothing
      In the expression:
        do let out = ...
           sc <- mkSharedContext
           C.scLoadPreludeModule sc
           C.scLoadCryptolModule sc
           ....
    |
104 |                     Nothing C.PublicAndPrivate Nothing
    |                             ^^^^^^^^^^^^^^^^^^

saw-tools/css/Main.hs:104:48: error: [GHC-83865]
    • Couldn't match expected type ‘C.ImportVisibility’
                  with actual type ‘Maybe a0’
    • In the sixth argument of ‘C.importCryptolModule’, namely
        ‘Nothing’
      In a stmt of a 'do' block:
        cryenv' <- C.importCryptolModule
                     sc cryenv (Left inputModule) Nothing C.PublicAndPrivate Nothing
      In the expression:
        do let out = ...
           sc <- mkSharedContext
           C.scLoadPreludeModule sc
           C.scLoadCryptolModule sc
           ....
    |
104 |                     Nothing C.PublicAndPrivate Nothing
    |                                                ^^^^^^^
```

--- a/build.sh
+++ b/build.sh
@@ -86,7 +86,7 @@ tgt_build() {
     cabal build $JOBSOPT \
                 exe:cryptol exe:cryptol-remote-api exe:cryptol-eval-server \
                 exe:saw exe:saw-remote-api \
-                exe:crux-mir-comp exe:extcore-info exe:verif-viewer \
+                exe:crux-mir-comp exe:css exe:extcore-info exe:verif-viewer \
                 test-suite:integration-tests test-suite:saw-core-tests \
                 test-suite:crux-mir-comp-tests \
                 test-suite:cryptol-saw-core-tests \

--- a/saw-tools/css/Main.hs
+++ b/saw-tools/css/Main.hs
@@ -101,7 +101,7 @@ cssMain css [inputModule,name] | cssMode css == NormalMode = do
     let ?fileReader = BS.readFile
     cryenv <- C.initCryptolEnv sc
     cryenv' <- C.importCryptolModule sc cryenv (Left inputModule)
-                    Nothing C.PublicAndPrivate Nothing
+                    Nothing False C.PublicAndPrivate Nothing
     processSymbol sc cryenv' out name
 
 cssMain css _ | cssMode css == VersionMode = do

